docs/getting_started.md

```diff
@@ -234,7 +234,7 @@ localhost                  : ok=0    changed=0    unreachable=0    failed=0
 ```
 
 ##### Provision - Creating a deployment config
-At the minimum, our APB should deploy the application [pods](https://docs.openshift.org/latest/architecture/core_concepts/pods_and_services.html#pods).  We'll do this by specifying a [deployment config](https://docs.openshift.org/latest/architecture/core_concepts/deployments.html#deployments-and-deployment-configurations).  One of the first tasks that's commented out on the in the `provision-my-test-apb/tasks/main.yml` is the creation of the deployment config.  You can uncomment it or paste the following.  Normally, you would replace the `image:` value with your own application image and
+At the minimum, our APB should deploy the application [pods](https://docs.openshift.org/latest/architecture/core_concepts/pods_and_services.html#pods).  We'll do this by specifying a [deployment config](https://docs.openshift.org/latest/architecture/core_concepts/deployments.html#deployments-and-deployment-configurations).  One of the first tasks that's commented out in the `provision-my-test-apb/tasks/main.yml` is the creation of the deployment config.  You can uncomment it or paste the following.  Normally, you would replace the `image:` value with your own application image and
```

Comment:
Not sure why github is failing to highlight the difference, so here it is:  s/on the in the/in the/